### DISCUSSION TITLE
VDH not working in Linux 22,04 firefox 133 from snap

### DISCUSSION THREAD

**Discussion by gsbgsb2** [aclap-dev/video-downloadhelper#1521] [Help and Q&A] [answered]
### Your question

Until last week, I was using Firefox version 122 on Linux 22.04 and the vdhcoapp worked one day, and one day it didn't. I used the recommendations on the help page, reinstalled the app, authorized its use, and it worked normally. Since Firefox updated to version 133, the app stopped working. Even though I follow the instructions on the help page, the app is not recognized in any way.

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: es-AR
coapp: {"found":false,"error":"No such native application net.downloadhelper.coapp"}
license: {"unneeded":true}
platform: x86-64 linux
UA: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:133.0) Gecko/20100101 Firefox/133.0
{
    "used_history_button": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 4
}

{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "/opt/vdhcoapp/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "linux",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/home/claudio",
  "converterBinary": "/opt/vdhcoapp/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}

**Comment by Mattmjs** [comment]
There was at the time a problem with Firefox 122 + Ubuntu 22.04, you've updated to Firefox 133 but still on 22.04.
There was also a recommendation to update Snap Core if you haven't already done this and see if it helps.

**Reply by gsbgsb2** [reply]
First of all, thanks for your reply, but I couldn't achieve anything with your suggestion. The problem is that Firefox was updated from version 122 to 133 without me noticing, and with it, the netvideo app... stopped working from one day to the next. And none of the solutions proposed on the help page seem to work in the new version. On the other hand, Snap is apparently in its latest version and I couldn't achieve anything on that front.

**Reply by Mattmjs** [reply, marked as answer]
Did you install the companion app manually or did you install it through the terminal ? Other than that I don't know why it's
not working for you. Also do you need Firefox installed as a snap app ? An alternative would be to install Firefox as a non snap version on your Ubuntu system and it would work no problem. There is some procedure to do that.

**Reply by gsbgsb2** [reply]
Thx for the advice, work very well.
